indra/llcommon/llprocessor.cpp

```diff
@@ -746,7 +746,7 @@ class LLProcessorInfoDarwinImpl : public LLProcessorInfoImpl
 		__cpuid(0x1, eax, ebx, ecx, edx);
 		if(feature_infos[0] != (S32)edx)
 		{
-			LL_ERRS() << "machdep.cpu.feature_bits doesn't match expected cpuid result!" << LL_ENDL;
+			LL_WARNS() << "machdep.cpu.feature_bits doesn't match expected cpuid result!" << LL_ENDL;
```

Comment:
I admit I'm curious about the circumstances under which this fails, and why it's in this particular PR?

Reply:
I've been running with this workaround locally for a while now when building on Apple Silicon and running unit tests under rosetta2.  I hoped to get it into the actions branch in advance of github making Apple Silicon M1 runner hosts available shortly so we can use them.  It is unrelated to the other changes here, but will be needed once we have M1 based build runners.

However, I don't know what the right fix is here.  I can't yet find the docs for apple's machdep.cpu.feature_bits contents, but it is unsurprising to me that this will have a new unexpected value when running under rosetta2.